From 65e052e659e373dc14b4b2bab3044dd471dd313c Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Thu, 20 Feb 2014 04:33:48 -0500
Subject: [PATCH] allow postfix-script read /var

Fix avc denied issue:

type=AVC msg=audit(1392887181.509:58): avc:  denied  \
{ read } for  pid=594 comm="postfix-script" name="var" dev="hda" \
ino=27745 scontext=system_u:system_r:postfix_master_t:s0-s15:c0.c1023 \
tcontext=system_u:object_r:var_t:s0 tclass=dir

Upstream-Status: pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 policy/modules/contrib/postfix.te |    2 ++
 1 file changed, 2 insertions(+)

--- refpolicy.orig/policy/modules/contrib/postfix.te
+++ refpolicy/policy/modules/contrib/postfix.te
@@ -221,6 +221,8 @@ allow postfix_master_t { postfix_postdro
 
 allow postfix_master_t postfix_prng_t:file rw_file_perms;
 
+allow postfix_master_t var_t:dir read;
+
 manage_dirs_pattern(postfix_master_t, postfix_spool_t, postfix_spool_t)
 manage_files_pattern(postfix_master_t, postfix_spool_t, postfix_spool_t)
 files_spool_filetrans(postfix_master_t, postfix_spool_t, dir)
